src/server/utils/circuitBreaker.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/* global BigInt */
+
+import { promisify } from 'util';
+import CircuitBreaker from 'opossum';
+import PrometheusMetrics from 'opossum-prometheus';
+import { register } from 'prom-client';
+import { composeModules, getModule } from 'holocron';
+import { getServerStateConfig } from './stateConfig';
+
+const immediate = promisify(setImmediate);
+const { rootModuleName } = getServerStateConfig();
+let eventLoopLagThreshold = 30;
+
+export const setEventLoopLagThreshold = (n) => {
+  eventLoopLagThreshold = Number(n) || 30;
```

Comment:
@mcollina what are your thoughts on this default value?

Reply:
30ms is _very_ aggressive. I would increase quite a bit. The 100-500ms range seems more ok for an SSR app (most React rendering go up to 80-100ms).